packages/console/src/containers/AppContent/components/Topbar/TenantSelector/index.tsx

```diff
@@ -107,6 +107,7 @@ function TenantSelector() {
           if (tenant) {
             toast.success(t('tenants.tenant_created', { name: tenant.name }));
             void mutate();
+            window.location.assign(new URL(`/${tenant.id}`, window.location.origin).toString());
```

Comment:
Can we use `navigate` of `react-router`?

Reply:
We've set basename for this app and the basename comes with current tenant id. We can not use navigate to switch to another tenant id, its out of the scope.